_posts/en/newsletters/2024-08-30-newsletter.md

```diff
@@ -0,0 +1,102 @@
+---
+title: 'Bitcoin Optech Newsletter #318'
+permalink: /en/newsletters/2024/08/30/
+name: 2024-08-30-newsletter
+slug: 2024-08-30-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+_No significant news this week was found in any of our [sources][]._
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Can a BIP152 compact block be sent before validation by a node that doesn't know all transactions?]({{bse}}123858)
+  Antoine Poinsot points out that forwarding [compact blocks][topic compact
+  block relay] before validating all of the included transactions would be a
+  denial-of-service vector.
```

Comment:
This wording suggests that the transactions need to be fully validated, but it's only necessary to validate that the transactions are committed to by the block header.

```suggestion
- [Can a BIP152 compact block be sent before validation by a node that doesn't know all transactions?]({{bse}}123858)
  Antoine Poinsot points out that forwarding [compact blocks][topic compact
  block relay] before validating all of the included transactions are committed
  to by the block header would be a denial-of-service vector.
```

Reply:
@vostrnad youre right, I understood this incorrectly, thanks!